### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daGscheid**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-## Hi there 👋
+Welcome to Stereum's Github Page 👋
+The Stereum team develops tools to manage the process of setting up & maintaining an Ethereum node simpler without compromising self sovereignty, privacy and flexibility.
+
+An Ethereum node enable its user to use dApps in the most decentralized fashion, to participate in the network by staking $ETH & to obtain/collect their own blockchain data. Stereum aims to cover your use-case as well!
```
Comment (stefa2k) [inline profile/README.md]:
```
An Ethereum node enables its user to interact with dApps in the most decentralized fashion, to participate in the network by staking Ether & to obtain/collect their own blockchain data. Don't trust, verify. Stereum aims to cover your use-case as well!
```

File: profile/README.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+
+🍄 By using Stereum it is simple to join Ethereum! ⯁
+If you are interested in following our development progress, check out our socials & donate to our GitCoin:
```
Comment (stefa2k) [inline profile/README.md]:
```
Follow our development progress, check out our socials & donate to our GitCoin:
```

File: profile/README.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+
+🌐 Website: stereum.net
```
Comment (stefa2k) [inline profile/README.md]: Please add links to all of the contacts: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

**Review by stefa2k — CHANGES_REQUESTED** [review]

**Review by stefa2k — APPROVED** [review]
👍

**Review by daGscheid — COMMENTED** [review]
Changes applied. Found the problem why I couldn't link in MD (emojis don't break lines)

**Pull request merged**